src/components/Footer.tsx

```diff
@@ -0,0 +1,27 @@
+import { useTranslations } from 'next-intl';
+
+export default function Footer() {
+  const t = useTranslations('Footer');
+  return (
+    <div>
+      <div>
+        <footer className="bg-black py-2 d-flex flex-wrap">
```

Comment:
Lets change the background from `bg-black` to `bg-primary`. I think the Karlsruhe-Green would fit the look of the page better.